PokeBuddy/settings.py

```diff
@@ -82,8 +82,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': "postgres",
+        "USER": "postgres",
+        "PASSWORD": "postgres",
+        "HOST": "db", # set in docker-compose.yml
+        "PORT": 5432, # default PostgreSQL port
```

Comment:
Will need to look into having these settings in a config file, rather than be statically set in the `settings.py` file.

Reply:
Opened #3 for this.